hydra-cluster/src/CardanoNode.hs

```diff
@@ -153,11 +166,19 @@ withBFTNode tracer cfg action = do
     traceWith tracer $ MsgSocketIsReady socket
     action rn
  where
-  dlgCertFilename i = "delegation-cert.00" <> show (i - 1) <> ".json"
-  signKeyFilename i = "delegate-keys.00" <> show (i - 1) <> ".key"
-  vrfKeyFilename i = "delegate" <> show i <> ".vrf.skey"
-  kesKeyFilename i = "delegate" <> show i <> ".kes.skey"
-  opCertFilename i = "opcert" <> show i <> ".cert"
+  dirname i =
+    "stake-pool-" <> show i
+
+  dlgCertFilename i =
+    dirname i </> "byron-delegation.cert"
+  signKeyFilename i =
+    dirname i </> "byron-delegate.key"
+  vrfKeyFilename i =
+    dirname i </> "vrf.skey"
+  kesKeyFilename i =
+    dirname i </> "kes.skey"
+  opCertFilename i =
+    dirname i </> "opcert.cert"
```

Comment:
We can get rid of all those `i`s by simply defining `dirname = "stake-pool-" <> show (nodeId cfg)`

Reply:
Addressed this myself.